README.rst

```diff
@@ -973,6 +973,18 @@ container), you must pass the --publish-ports argument to buildrunner.
 This must never be used on a shared server such as a build server as it could
 cause port mapping conflicts.
 
+Image Security Scans
+====================
+
+Pushed docker images may be automatically scanned for vulnerabilities using the ``security-scan``
+global configuration options or the ``--security-scan-*`` command line options that may override
+the global configuration.  Just set ``security-scan.enabled`` to true to enable automatic scans.
+
+The ``max-score-threshold`` may also be configured to fail the build if the max score of the
+detected vulnerabilities is greater than or equal to the ``max-score-threshold`` value.
```

Comment:
What is the range of valid values for the max-score?

Reply:
It would be helpful to have that range in the docs.

Reply:
I'm not sure on the *actual* possible range, and I figured if they set it low (i.e. negative, 0), they will always hit it, so I just left it open since the functionality is intact in that case as well.